event_generation/locustfile.py

```diff
@@ -0,0 +1,36 @@
+from datetime import datetime
+import random
+import json
+
+from locust import User, task, between
+import boto3
+
+import config as cfg
+from tickers import Tickers
+
+
+class StockUser(User):
```

Comment:
Aun siendo User propio de la libreria de locust, hace realmente falta pasarlo como parametro?? Desde el desconocimiento, lo usa internamente para algo o es omitible? ya que se usa a la hora de definir la clase pero al referenciarla, unicamente se hace llamando a la funcion get_data, que basicamente es sacar valores random de las 3 empresas y no tiene ningun tipo de impacto a la hora de conectarse al stream de kinesis.

Reply:
User its the class from which StockUser inherits, it is necessary for the script to user Locust. See [docs](https://docs.locust.io/en/stable/writing-a-locustfile.html#user-class)